File: Basic/problem_17.c
/*
Write a program that accepts a number from a user and prints the factorial of that number
*/
// solved by Ashwani2529
#include<stdio.h>  
int main()    
{    
 int i,fact=1,number;    
 printf("Enter a number: ");    
  scanf("%d",&number);    
    for(i=1;i<=number;i++){    
      fact=fact*i;    
  }       
return fact;  
}   

File: Basic/Python/problem_17.py
# 17. Write a program that accepts a number from a user and prints the factorial of that number
# (the range of the answer should be greater than 4294967295)
# by Nitin Kumar

n = int(input('Enter a number:'))
x = lambda num : 1 if num <= 1 else num*x(num-1)
print('{} ! = {}'.format(n, x(n)))
